client/lib/gsuite/new-users.ts

```diff
@@ -27,6 +27,7 @@ export interface GSuiteProductUser {
 	lastname: string;
 	email: string;
 	password: string;
+	passwordResetEmail?: string;
```

Comment:
One additional note - It would be great to follow the same naming convention between the frontend and the backend 🙏. Right now we refer to this field as `password reset email` in Calypso and `recovery email` in the backend. I would be tempted to use the latter as it is shorter but I don't have a strong opinion about this.